c/src/ml-api-internal.h

```diff
@@ -69,24 +69,30 @@ typedef enum
   SUPPORTED = 1
 } feature_state_t;
 
+/**
+ * @brief Enumeration for machine_learning feature.
+ */
+typedef enum {
+  ML_INF_FEATURE = 0,
```

Comment:
recommend: name rule and nntrrainer ML_FEATURE_INFERENCE / ML_FEATURE_TRAINING / ML_FEATURE_SERVICE

Reply:
I agreed with @jaeyun-jung 's opinion.